Source/WebCore/platform/audio/SincResampler.cpp

```diff
@@ -293,6 +293,7 @@ void SincResampler::process(std::span<float> destination, size_t framesToProcess
     }
 }
 
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
```

Comment:
This incorrectly covers the `#if USE(ACCELERATE)` which doesn't warn and is used by Apple ports. We should not silence warnings in valid code.

This comment likely applies elsewhere.

Reply:
Right, I will double-check when rebasing to make sure there aren't cases like this left.